src/js/flows/saveWindowState.test.js

```diff
@@ -0,0 +1,12 @@
+/* @flow */
+import initTestStore from "../test/initTestStore"
+import saveWindowState from "./saveWindowState"
+
+test("does not mutate state", () => {
+  let store = initTestStore()
+  let prevState = {...store.getState()}
+
+  store.dispatch(saveWindowState())
+
+  expect(store.getState()).toEqual(prevState)
```

Comment:
Before the fix above, this would fail with the "errors" "notice" and "handlers" keys missing.